monai/data/image_writer.py

```
@@ -116,7 +116,22 @@ def resolve_writer(ext_name, error_if_not_found=True) -> Sequence:
+            "mhd": "itk",
+            "nrrd": "itk",
+            "png": "pillow",
+            "jpg": "pillow",
+            "jpeg": "pillow",
+            "tif": "pillow",
+            "tiff": "pillow",
+            "bmp": "pillow",
+        }
+        hint = install_hints.get(fmt)
+        extra = f" Try installing the required package: pip install {hint}" if hint else ""
+        raise OptionalImportError(f"No ImageWriter backend found for '{fmt}'.{extra}")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)